playground/frontend/lib/constants/assets.dart

```diff
@@ -23,6 +23,7 @@ const kOutputRightIconAsset = 'output_right.svg';
 const kOutputLeftIconAsset = 'output_left.svg';
 const kShortcutsIconAsset = 'shortcuts.svg';
 const kGithubIconAsset = 'github.svg';
+const kColabIconAsset = 'colab.svg';
```

Comment:
On some (unmerged?) branch we were dropping these constants in favor of flutter_gen. Can this PR be done on that branch?

Reply:
I don't think, that it's a good idea, because in this PR we are depends on colab link code on backend. So if we try to create PR on that branch with this code, where will be a lot of additional changes. I think it will be better to resolve conflicts

Reply:
Colab field is in `master` already, so you may try. But I am not pushing.